pandas/core/indexes/base.py

```diff
@@ -820,20 +820,22 @@ def repeat(self, repeats, axis=None):
     # --------------------------------------------------------------------
     # Copying Methods
 
-    def copy(self, name=None, deep=False, dtype=None, **kwargs):
+    def copy(self, name=None, deep=False, dtype=None, names=None):
         """
         Make a copy of this object.  Name and dtype sets those attributes on
         the new object.
 
         Parameters
         ----------
-        name : str, optional
+        name : Optional[Hashable]
```

Comment:
should we call this Label (from pandas._typing)?

Reply:
Ok, I've changed this to Label.